Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorEffects/SegmentEditorLogicalEffect.py

```diff
@@ -41,35 +43,36 @@ def helpText(self):
 <li><b>Clear:</b> clears selected segment.</li>
 <li><b>Fill:</b> completely fills selected segment.</li>
 </ul><p>
-<b>Selected segment:</b> segment selected in the segment list - above. <b>Modifier segment:</b> segment chosen in segment list in effect options - below.
-<p></html>"""
+<b>Selected segment:</b> segment selected in the segment list - above. <b>Modifier segment:</b> segment chosen in
+segment list in effect options - below.
+<p></html>""")
```

Comment:
```suggestion
</p></html>""")
```

Reply:
I would vote for keeping the text simpler and omit the optional paragraph closing tag (unless there is a specific reason to add it).

Reply:
Closing tag may indeed be omitted in a certain situation. See https://html.spec.whatwg.org/multipage/syntax.html#syntax-tag-omission

The proposed simplification makes sense.